libraries/ts-command-line/src/parameters/CommandLineDefinition.ts

```diff
@@ -49,7 +49,9 @@ export interface IBaseCommandLineDefinition {
    *
    * @remarks
    * The environment variable name must consist only of upper-case letters, numbers,
-   * and underscores. It may not start with a number.
+   * and underscores. It may not start with a number. To disable this validation, you
+   * must set `{@link IBaseCommandLineDefinition.allowNonStandardEnvironmentVariable}`
```

Comment:
```suggestion
   * and underscores. It may not start with a number. To disable this validation,
   * set `{@link IBaseCommandLineDefinition.allowNonStandardEnvironmentVariable}`
```